docs/mkdocs/docs/api/macros/json_diagnostics.md

```diff
@@ -32,6 +32,11 @@ When the macro is not defined, the library will define it to its default value.
     across different compilation units: `basic_json` objects with differently defined `JSON_DIAGNOSTICS` macros are
     not compatible!
 
+!!! hint "CMake option"
+
+    The CMake option [`JSON_Diagnostics`](../../integration/cmake.md#json_diagnostics)
+    can be used to set this macro at build-time.
```

Comment:
Wouldn't this be at "configure time"?

Reply:
Looks like I forgot to adopt the new wording I used everywhere else.